content/md/products-super-power/improve-data-quality.md

```diff
@@ -69,16 +69,18 @@ To help you even further, in the `Attributes` panel, you can select spelling cor
 ::: info
 Spell check is available for the English, French, German, Spanish, Italian, Dutch, Danish, Swedish, Norwegian and Portuguese (Brazilian) locales.
 Spell check is usable with localizable text and text area attributes.
-:::
+If you use a single locale, then spell-check are usable on text and text area attributes.
 
-::: info
 Title formatting suggestion is available for English locales.
-:::
 
-::: info
 Even if they don't have permission to edit, all PIM users will be able to read data quality insights on all attributes.
 :::
 
+From the `data quality panel`, you can view attribute labels and options with spelling mistakes. By clicking on the link, you are led to the **settings** screen from which you can correct these mistakes hence improving your consistency grade.
+
+![Image data quality spellcheck on attribute](../img/data-quality-attribue-spelling.gif)
```

Comment:
Is it a new image? I don't see it in the list of files joined to this pull request.

Reply:
no, it's not a new image.